checklists/lz_checklist.en.json

```diff
@@ -491,7 +491,7 @@
       "text": "If you need transit between ExpressRoute and VPN gateways, use Azure Route Server.",
       "guid": "ce463dbb-bc8a-4c2a-aebc-92a43da1dae2",
       "severity": "Medium",
-      "link": "https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager#to-enable-transit-routing-between-expressroute-and-azure-vpn"
+      "link": "https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager"

```

Comment:
Would you be able to update your branch to the previous link? https://docs.microsoft.com/azure/expressroute/expressroute-howto-coexist-resource-manager#to-enable-transit-routing-between-expressroute-and-azure-vpn

Reply:
Should be fixed now - thank you for spotting this.